docs/cluster_spec.md

```diff
@@ -912,6 +912,45 @@ spec:
 
 The above will result in the flags `--kube-reserved=cpu=1,memory=2Gi,ephemeral-storage=1Gi --kube-reserved-cgroup=/kube-reserved --kubelet-cgroups=/kube-reserved --runtime-cgroups=/kube-reserved --system-reserved=cpu=500m,memory=1Gi,ephemeral-storage=1Gi --system-reserved-cgroup=/system-reserved --enforce-node-allocatable=pods,system-reserved,kube-reserved` being added to the kubelet.
 
+This assumes that the `kube-reserved` and `system-reserved` cgroups exist on the node prior to kubelet start. Depending on your node image, these may not exist and hence prevent the kubelet from starting. To ensure these exist, you can provision these cgroups with the following additions to each `InstanceGroup` spec:
```

Comment:
This page is specifically for the `Cluster` resource. Maybe move this to `https://kops.sigs.k8s.io/instance_groups/` and reference on this page?